static-site/content/docs/07-visualisation/02-map-interpretation.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "How to interpret what's shown on the map"
+anchorText: "Interpreting the Map"
+---
+
+The map is a centerpiece of Nextstrain's visualisation, able to convey complex information to a wide audience.
+This how-to is intended to provide an overview of what's being displayed, and hopefully dispel some potential misinterpretations.
+
+
+![](./img/labelled-map.png)
+_Figure 1. [Nextstrain map](https://nextstrain.org/zika) of Zika virus movement between countries. Numerical labels are referred to in the text._
+
+
+Before we begin, it's important to keep in mind that all the data displayed on the map is drawn directly from the phylogenetic tree.
+In practice, every dataset is subject to some form of sampling bias, and therefore so will the map.
+If there are no samples (genomes) from a certain location in the dataset being visualised, then we have no data to displaye on our map!
+
+
+As you **interact with Nextstrain** -- whether you zoom into the tree, filter the data, change the color-by or geographic resolution -- then every panel, including the map, will update to represent just the selected data under the desired view settings.
+
+
+## Demes (Circles)
+
+Perhaps the most fundamental aspect of Nextstrain's map, the circles (demes) associate samples (tips on the tree) with locations on the map.
+The demes use different visual techniques to convey different pieces of information:
+
+![](./img/labelled-deme.png)
+_Figure 2. Examples of demes.
+B,C and D are representations of samples from the USA in [the Zika dataset](https://nextstrain.org/zika) with different choices of color-by._
+
+
+The **size (area) of the circle** is used to convey the number of samples associated with the current location.
```

Comment:
Will this still be true once the latest changes from Richard are merged in? I think so, but wanted to check. If not, I think we can just be a little more vague and it will be fine :)

Reply:
Those changes are already up on nextstrain.org so I tried to use language to indicate that a deme's size "should only be interpreted relative to other circles currently displayed on the map". 

I think the only way to do this going forward is to add a dynamic legend showing the sizes.

Reply:
I think this is good!